auto_landing/src/tracking.cpp

```diff
@@ -0,0 +1,50 @@
+#include<auto_landing/tracking.hpp>
+
+
+namespace ariitk::auto_landing {
+
+ void Tracking::init(ros::NodeHandle& nh) {
+
+        set_firefly_pose_ = nh.advertise<geometry_msgs::PoseStamped>("/firefly/command/pose", 1);
+        husky_pose_ = nh.subscribe("/husky_velocity_controller/odom", 1, &Tracking::poseClbk,this);
+        landing_server_ = nh.advertiseService("to_land", &Tracking::serverClbk,this);
```

Comment:
Also, you are not supposed to advertise this service. Pranav's node will advertise the service, you will just call it. So you need to only keep the service client, not service server.